docs/guides/platforms/eks-irsa.md

```diff
@@ -0,0 +1,607 @@
+---
+title: EKS | Stash
+description: Using Stash in Amazon EKS
+menu:
+  docs_{{ .version }}:
+    identifier: platforms-eks-irsa
+    name: EKS IRSA
+    parent: platforms
+    weight: 10
+product_name: stash
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+# Using Stash with Amazon EKS
+
+This guide will show you how to use Stash to backup and restore a KubeDB database running in [Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/). Here, we are going to backup a MariaDB database into [AWS S3 bucket](https://aws.amazon.com/s3/) using IRSA(IAM Roles for Service Accounts). Then, we are going to show how to restore this backed up data.
```

Comment:
a KubeDB database -> a database -> a MariaDB database

Reply:
This guide will show you how to use IRSA(IAM Roles for Service Accounts) of [Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/) with Stash. Here, we are going to backup a database and store the backed up data into a [AWS S3 bucket](https://aws.amazon.com/s3/).